torchci/components/JobConclusion.module.css

```diff
@@ -56,9 +56,11 @@
 }
 
 .flaky {
-  color: lightgreen;
+  font-size: 1.3em;
+  color: rgba(13, 169, 13, 0.797);
```

Comment:
Why are we adding transparency there? And if possible it would be great for consistency sake to stick either to hex or names